src/cmd/create.go

```diff
@@ -0,0 +1,490 @@
+		command = append(command, "--mnt-link")
+	} else {
+		mntBind := []string{"--volume", "/mnt:/mnt:rslave"}
+		createArgs = append(createArgs, mntBind...)
+	}
+
+	if utils.PathExists("/run/media") {
+		runMediaBind := []string{"--volume", "/run/media:/run/media:rslave"}
+		createArgs = append(createArgs, runMediaBind...)
+	}
+
+	logrus.Info("Checking if /usr is mounted read-only or read-write")
+	usrMountPoint, err := utils.GetMountPoint("/usr")
+	if err != nil {
+		logrus.Error(err)
+		logrus.Fatal("Failed to get the mount-point of /usr")
+	}
+	logrus.Infof("Mount-point of /usr is %s", usrMountPoint)
+	usrMountSourceFlags, err = utils.GetMountOptions(usrMountPoint)
+	if err != nil {
+		logrus.Error(err)
+		logrus.Fatalf("Failed to get the mount options of %s", usrMountPoint)
+	}
+	logrus.Infof("Mount flags of /usr on the host are %s", usrMountSourceFlags)
+
+	if !strings.Contains(usrMountSourceFlags, "ro") {
+		usrMountDestinationFlags = "rw"
+	}
+
+	// Inside of a toolbox we want to be able to access dbus for using flatpak-spawn and for users, who work with dbus.
+	logrus.Info("Preparing dbus system bus address")
+	dbusSystemBusAddress = strings.Split(viper.GetString("DBUS_SYSTEM_BUS_ADDRESS"), "=")[1]
+	dbusSystemBusAddress, err = filepath.EvalSymlinks(dbusSystemBusAddress)
+	if err != nil {
+		logrus.Error(err)
+	}
+
+	dbusSystemBusAddressBind := []string{"--volume", fmt.Sprintf("%s:%s", dbusSystemBusAddress, dbusSystemBusAddress)}
+	createArgs = append(createArgs, dbusSystemBusAddressBind...)
+
+	logrus.Info("Preparing sssd-kcm socket")
+	args = []string{"show", "--value", "--property", "Listen", "sssd-kcm.socket"}
+	output, err := utils.SystemctlOutput(args...)
+	if err != nil {
+		logrus.Error("Failed to use 'systemctl show'")
+	}
+
+	kcmSocket = strings.Trim(string(output), "\n")
+
+	if kcmSocket == "" {
+		logrus.Error("Failed to read property Listen from sssd-kcm.socket")
+	} else {
+		logrus.Infof("Checking value %s of property Listen in sssd-kcm.socket", kcmSocket)
+		if !strings.Contains(kcmSocket, " (Stream)") {
+			kcmSocket = ""
+			logrus.Error("Unknown socket in sssd-kcm.socket\nExpected SOCK_STREAM")
+		}
+		if !strings.Contains(kcmSocket, "/") {
+			kcmSocket = ""
+			logrus.Error("Unknown socket in sssd-kcm.socket\nExpected file system socket in the AF_UNIX family")
+		}
+	}
+
+	logrus.Infof("Parsing value %s of property Listen in sssd-kcm.socket", kcmSocket)
+	if kcmSocket != "" {
+		kcmSocket = strings.TrimSuffix(kcmSocket, " (Stream)")
+		kcmSocketBind = []string{"--volume", fmt.Sprintf("%s:%s", kcmSocket, kcmSocket)}
+		createArgs = append(createArgs, kcmSocketBind...)
+	}
+
+	logrus.Info("Checking if 'podman create' supports option '--ulimit host'")
+	if podman.CheckVersion("1.5.0") {
+		logrus.Info("Option '--ulimit host' is supported")
+		ulimitHost = []string{"--ulimit", "host"}
+	} else {
+		logrus.Info("Option '--ulimit host' is not supported")
+	}
+
+	homeEnv := strings.Split(viper.GetString("HOME"), "=")[0]
```

Comment:
Is the `strings.Split` really necessary? A few lines above, it's not split.

Either way, we could use `user.Current().HomeDir`, can't we? We already query `user.Current()` for the UID.